config/chime_keep_gpu_warm.yaml

```
@@ -0,0 +1,3792 @@
+  metadata_pool: main_pool
+  rfi_valve_buffer_0:
+    kotekan_buffer: standard
+  rfi_valve_buffer_1:
+    kotekan_buffer: standard
+  rfi_valve_buffer_2:
+    kotekan_buffer: standard
+  rfi_valve_buffer_3:
+    kotekan_buffer: standard
+
+gpu_rfi_mask_output_buffers:
+  num_frames: buffer_depth
+  frame_size: num_local_freq * samples_per_data_set / sk_step
+  metadata_pool: main_pool
+  gpu_rfi_mask_output_buffer_0:
+    kotekan_buffer: standard
+  gpu_rfi_mask_output_buffer_1:
+    kotekan_buffer: standard
+  gpu_rfi_mask_output_buffer_2:
+    kotekan_buffer: standard
+  gpu_rfi_mask_output_buffer_3:
+    kotekan_buffer: standard
+
+gpu_rfi_bad_input_buffers:
+  num_frames: buffer_depth
+  frame_size: sizeof_float * num_elements * num_local_freq
+  metadata_pool: main_pool
+  gpu_rfi_bad_input_buffer_0:
+    kotekan_buffer: standard
+  gpu_rfi_bad_input_buffer_1:
+    kotekan_buffer: standard
+  gpu_rfi_bad_input_buffer_2:
+    kotekan_buffer: standard
+  gpu_rfi_bad_input_buffer_3:
+    kotekan_buffer: standard
+
+# Bad inputs buffer
+bad_input_buffer:
+  num_frames: buffer_depth
+  frame_size: num_elements
+  metadata_pool: main_pool
+  bad_inputs_buffer:
+    kotekan_buffer: standard
+
+rfi_var_output_buffers:
+  num_frames: buffer_depth * 50
+  frame_size: num_local_freq * samples_per_data_set / sk_step * sizeof_float
+  metadata_pool: main_pool
+  gpu_rfi_var_output_buffer_0:
+    kotekan_buffer: standard
+  gpu_rfi_var_output_buffer_1:
+    kotekan_buffer: standard
+  gpu_rfi_var_output_buffer_2:
+    kotekan_buffer: standard
+  gpu_rfi_var_output_buffer_3:
+    kotekan_buffer: standard
+
+lost_samples_buffer:
+  kotekan_buffer: standard
+  num_frames: 2 * buffer_depth
+  frame_size: samples_per_data_set
+  metadata_pool: main_pool
+
+rfi_compressed_lost_samples_buf:
+  kotekan_buffer: standard
+  num_frames: buffer_depth * 50
+  frame_size: sizeof_int * samples_per_data_set / sk_step
+  metadata_pool: main_pool
+
+rfi_compressed_lost_samples_buffers:
+  num_frames: buffer_depth * 2
+  frame_size: sizeof_int * samples_per_data_set / sk_step
+  metadata_pool: main_pool
+  rfi_compressed_lost_samples_buf_0:
+    kotekan_buffer: standard
+  rfi_compressed_lost_samples_buf_1:
+    kotekan_buffer: standard
+  rfi_compressed_lost_samples_buf_2:
+    kotekan_buffer: standard
+  rfi_compressed_lost_samples_buf_3:
+    kotekan_buffer: standard
+
+hfb_compressed_lost_samples_buffers:
+  num_frames: buffer_depth * 2
+  frame_size: sizeof_int * samples_per_data_set / (factor_upchan * downsample_time)
+  metadata_pool: main_pool
+  hfb_compressed_lost_samples_buf_0:
+    kotekan_buffer: standard
+  hfb_compressed_lost_samples_buf_1:
+    kotekan_buffer: standard
+  hfb_compressed_lost_samples_buf_2:
+    kotekan_buffer: standard
+  hfb_compressed_lost_samples_buf_3:
+    kotekan_buffer: standard
+
+
+##########################
+#### Pipeline Stages #####
+##########################
+gen_data:
+  type: random
```

Random will keep trying to generate a new array every time around, which will be too slow. 
```suggestion
  type: const
```